ui/src/app/kvm/components/KVMActionFormWrapper/KVMActionFormWrapper.test.tsx

```diff
@@ -5,7 +5,7 @@ import configureStore from "redux-mock-store";
 
 import KVMActionFormWrapper from "./KVMActionFormWrapper";
 
-import { PodAction, PodType } from "app/store/pod/types";
+import { PodFormNames, PodType } from "app/store/pod/types";
```

Comment:
I've changed the name of `PodAction` to `PodFormNames` because we need to move away from coupling actions to header forms.